mozregression/releases.py

```diff
@@ -59,8 +59,8 @@ def date_of_release(release):
     Provide the date of a release.
     """
     try:
-        return releases()[release]
-    except KeyError:
+        return releases()[int(release)]
+    except KeyError, ValueError:
```

Comment:
As stated in https://bugzilla.mozilla.org/show_bug.cgi?id=1214192, this is not doing what you want.